app/Family.php

```diff
@@ -176,18 +177,39 @@ public function getEntitlementAttribute()
         return $this->getStatus()['vouchers'];
     }
 
+    /**
+     * Gets the due date or Null;
+     *
+     * @return mixed
+     */
     public function getExpectingAttribute()
     {
-        // return true if there is a child that has a false 'born' attribute
-
-        $expecting = false;
+        $due = null;
```

Comment:
So if the baby is born - returns null

Reply:
Yup. it's supposed to find and return the due date if it exists.